builtin/logical/aws/secret_access_keys_test.go

```diff
@@ -40,3 +41,51 @@ func TestNormalizeDisplayName_NormNotRequired(t *testing.T) {
 		}
 	}
 }
+
+func TestGenUsername(t *testing.T) {
```

Comment:
Can we add a test case for the default template?

Reply:
Added